docs/customer_management/cp_page_builder.md

```diff
@@ -0,0 +1,140 @@
+
+## Configure Page Builder access to Customer Portal
+
+First, you need to create a root folder for Customer Portals, 
+its `location_id` will be later specified in the configuration as [a tree root](multisite_configuration.md#location-tree).
+To do it, go to **Content**->**Content structure** and add a new folder in the Content tree where you will be adding Customer Portals.
+
+![Customer Portals folder](img/cp_folder_for_portals.png)
+
+To be able to see Customer Portal site in the Page Builder you need to add `custom_portal` SiteAccess to the configuration.
+First, go to `config/packages/ibexa.yaml` and add `custom_portal` to
+the SiteAccess `list` and to `corporate_group`.
+
+Next add configuration for `corporate_group` and `custom_portal` under `system`.
+Remember to specify `location_id` of the root folder for Customer Portals, you will find it under **Details**. 
+
+```yaml hl_lines="8 12 14 16"
+ibexa:
+    siteaccess:
+        list:
+            - import
+            - site
+            - admin
+            - corporate
+            - custom_portal    
+        groups:
+            site_group: [import, site]
+            storefront_group: [site]
+            corporate_group: [corporate, custom_portal]
+    system:
+        corporate_group:
+            languages: [eng-GB]
+        custom_portal:
+            languages: [ eng-GB ]
+            content:
+                tree_root:
+                    location_id: location_id_of_customer_portals_root_folder
+                    excluded_uri_prefixes: [ /media, /images ]
+```
+
+Next, go to `config/packages/ibexa_admin_ui.yaml` and add `custom_portal` to the SiteAccess list available to Page Builder:
+
+```yaml
+ibexa:
+    system:
+        page_builder:
+            siteaccess_list:
+                - site
+                - corporate
+                - custom_portal
+```
+
+## Customer Portal container
+
+Now, you can go back to your **Customer Portals** root folder and select **Create content**.
+There you will see two possibilities **Customer Portal** and **Customer Portal Page**.
+
+The first one is a separate Content Type used as a container for your Customer Portal pages.
+Customer Portals containers should be used to sort Customer Portal pages and any other Content Types used by them, such as articles, inside the root folder.
+We highly recommend that you use them instead of folders to divide and store your portals.
+
+![Create content tab](img/cp_portal_vs_page.png)
+
+## Assigning Portal to Customer Group
+
+You can assign multiple Customer Portal containers or Pages to a specific Customer Group.
+First, you need to grant the following permissions to company members from the Customer Group: 
+
+- `user/login` to `custom_portal` SiteAccess
+- `content/read` to selected Customer Portals
+
+![Customer Portal permissions](img/cp_permissions.png)
+
+Next, add portal mapping configuration in `config/services.yaml`:
```

Comment:
That part is only important/required if you are planning to separate your portals by customer groups, it is not mandatory otherwise.

What is missing here is that you can specify your own custom logic for redirecting members to specific portals by implementing `\Ibexa\Contracts\CorporateAccount\CustomerPortal\PickRule\CustomerPortalPickRule` and taggin it with `ibexa.corporate_account.customer_portal.pick_rule`